test/test_util/custom_matchers.dart

```diff
@@ -55,7 +55,9 @@ class ClassNameMatcher extends Matcher {
   }
 
   @override
-  bool matches(String className, Map matchState) {
+  bool matches(className, Map matchState) {
+    className = className as String; // ignore: avoid_as
```

Comment:
Update from that issue. On Dart 1.20.1 you can use `@checked` to override param types.